Packs/PAN-OS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS",
     "description": "Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.",
```

Comment:
@mayagoldb @kirbles19.
Let's re think the desc, and also the tags, keywords and use cases, as this is a very comprehensive pack.